lib/proxy.js

```diff
@@ -50,8 +50,10 @@ const proxy = function(CONFIG) {
     function (req, res) {
       for (let [key, value] of routeMap) {
         const urlMatch = req.url.match(key);
-        if (urlMatch) {
-          fileSystem.createReadStream(req.url.replace(urlMatch[0], value).split('?')[0]).pipe(res);
+        const filePath = urlMatch && req.url.replace(urlMatch[0], value).split('?')[0];
+
+        if (urlMatch && fileSystem.existsSync(filePath)) {
```

Comment:
Should it be `if (filePath && fileSystem.existsSync(filePath)) {`?

Reply:
Why would we need to check for filePath? If it is `undefined` or `''` then fileSystem.existsSync will still return false.

Reply:
Well then the check should not include `urlMatch`. Something like this?
```
if (fileSystem.existsSync(filePath)) {
```

Reply:
correct, `urlMatch` is not required anymore